turborepo-tests/integration/fixtures/turbo-configs/package-task.json

```diff
@@ -1,7 +1,5 @@
 {
   "$schema": "https://turborepo.com/schema.json",
-  "globalDependencies": ["foo.txt"],
```

Comment:
We were using an invalid package configuration for testing that package task syntax wasn't used in the task list